apps/console/src/features/applications/components/application-list.tsx

```diff
@@ -64,6 +64,7 @@ import {
     ApplicationTemplateListItemInterface
 } from "../models";
 import { ApplicationTemplateManagementUtils } from "../utils";
+import { applicationListConfig } from "../../../extensions/configs/application-list";
```

Comment:
Lets fix these @pavinduLakshan can you assist?

Reply:
Sure

Reply:
Fixed with commit fbc9b4d354e56a75b1814bd110884160c12b6358